docs/manual/src/internals/ffi_converter_traits.md

```diff
@@ -0,0 +1,68 @@
+## Rust FFI conversion traits
+
+UniFFI leverages a set of FFI converter traits to implement [lifting and lowering](./lifting_and_lowering.md) on the Rust side.
+Each trait handles a single step in the lifting/lowering process (e.g. lifting an argument, lowering a return, etc.).
+We implement the trait for each type used in the exported API then use that trait in the codegen.
```

Comment:
Above it talks about multiple traits (but never names them), and this sentence now talks about _the trait_ (singular). I feel like that's a bit hard to understand without already knowing the details.
Maybe we can be a bit more specific here.

Reply:
Ah, I see the list of traits is further below. I still think this sentence could be worded slightly differently, I'll think about it